pkg/init/backend/interactive.go

```diff
@@ -2,13 +2,14 @@ package backend
 
 import (
 	"fmt"
+	"sort"
+	"strconv"
+
 	"github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
 	"github.com/devfile/library/pkg/devfile/parser"
 	parsercommon "github.com/devfile/library/pkg/devfile/parser/data/v2/common"
 	"github.com/fatih/color"
 	"github.com/redhat-developer/odo/pkg/log"
```

Comment:
Might as well group all the odo pkgs together.

Reply:
Will do - thanks.
Out of curiosity, is there a tool for doing such import grouping and sorting automagically?
When my IDE showed warnings, I thought `gofmt` would be able to correctly sort imports, but this was not the case. I had to rely on `goimports` instead.